modules/quickstart-public-cloud/pages/installation.adoc

```diff
@@ -28,5 +30,45 @@ When you have logged in to the administrator account for the first time, change
 
 
 Add the required software products to your client, and schedule a repository synchronization.
-You might also need to set up custom repositories and channels to manage {ubuntu} or {rhel} products.
-For more on custom channels, see xref:administration:custom-channels.adoc[].
+
+
+
+.Procedure: Adding the openSUSE Channels
+
+. At the command prompt on the {productname} Server, as root, install the [systemitem]``spacewalk-utils`` package:
++
+----
+zypper in spacewalk-utils
+----
+. Add the openSUSE channels.
++
+----
+spacewalk-common-channels \
+opensuse_leap15_1 \
+opensuse_leap15_1-non-oss \
+opensuse_leap15_1-non-oss-updates \
+opensuse_leap15_1-updates \
+opensuse_leap15_1-uyuni-client
+----
+
+. Synchronize the new custom channels.
+
+[IMPORTANT]
+====
+You need all the new channels fully synchronized before bootstrapping any openSUSE client.
+openSUSE channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+To synchronize, navigate to menu:Software[Manage > Channels], and for each channel you created, click it and:
+
+. Navigate to the [guimenu]``Repositories`` tab.
+. Navigate to the [guimenu]``Sync`` sub-tab, and click btn:[Sync Now] to begin synchronization immediately.
+
+You can also schedule synchronization from this screen.
+
+endif::[]
```

Comment:
This makes the build fail. Where is the matching `if[]` ?

Reply:
that was just copypasta